### PR TITLE
closes #3: add xpec test runner

### LINKED ISSUE #3: Integrate CI/CD for Initial Schematron
As an OSCAL user, to ensure ongoing development of validations I use in 18F/fedramp-automation#1 are actively tested, prepared, and published in a reliable way, I expect the validation developers to integrate with the existing continuous integration and continuous development pipelines in Github Actions upstream GSA developers use.

### PR DESCRIPTION
**Pull request by mike-stern**

### CODE REVIEW

File: .github/workflows/xpecRunner.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# This is a basic workflow to help you get started with Actions
+
+name: runxpec
```
Comment (ohsh6o) [inline .github/workflows/xpecRunner.yml:3]: It is XSpec, right? Also should we give it a friendly name [like the GSA/NIST devs do with OSCAL](https://github.com/18F/fedramp-automation/blob/master/.github/workflows/content-artifacts.yml#L13)? I was being old-school last time when I was worried about human-readable names and spaces, but it seems to work and make it more clear.

Should we make the intent a little more clear, like `OSCAL Validations: Unit Tests`?

**Review by ohsh6o — CHANGES_REQUESTED** [review]
Looks amazing! I only have two minor comments, and one higher-level question I wanted to surface here: are we going to add a step to save the intermediate XSL from XSpec and the final report to see the last part and also go back to see why it failed later?

File: .github/workflows/xpecRunner.yml (hunk 1 above)
Comment (mike-stern) [reply .github/workflows/xpecRunner.yml]: modified comments and job name to better reflect what we're doing

**Comment by mike-stern** [comment]
@ohsh6o  does the reporting (example [here](https://github.com/18F/fedramp-automation/pull/14/checks?check_run_id=1343102102) match what you would expect? 

I am not retaining the xsl, but the build breaks and the unit test outputs are now visible as part of the action.

Also, I accepted the wildcarding for on push and pr, but am getting 3 triggers of the actions on each commit.

**Comment by ohsh6o** [comment]
> @ohsh6o does the reporting (example [here](https://github.com/18F/fedramp-automation/pull/14/checks?check_run_id=1343102102) match what you would expect?

Oh you tried doing something more elaborate than I had planned, I was suggesting [like here we store the outputs next to the action](https://github.com/ohsh6o/skyrim-schematron-tutorial/blob/main/.github/workflows/report.yml#L65-L74) ([an example job with output](https://github.com/ohsh6o/skyrim-schematron-tutorial/actions/runs/322546241)), not even in the repo directly and committing it back, which you are doing, but that is pretty neat!

> I am not retaining the xsl, but the build breaks and the unit test outputs are now visible as part of the action.

That's fine!

> Also, I accepted the wildcarding for on push and pr, but am getting 3 triggers of the actions on each commit.

Yeah, maybe we should look at this together, maybe we need one or the other (`push` or `pull_request`, but not both). I guess I/we need to read up!

**Review by ohsh6o — APPROVED** [review]

**Pull request merged**